main.tf

```diff
@@ -0,0 +1,12 @@
+
+resource "null_resource" "ansible_provisioner" {
```

Comment:
I think we need this `null_resource` to depend on another `null_resource` that does a `remote-exec`  on `/bin/true` and waits for the machine to come online.